src/app/containers/Image/index.jsx

```diff
@@ -2,16 +2,26 @@ import React from 'react';
 import { filterForBlockType } from '../../helpers/blockHandlers';
 import { imageModelPropTypes } from '../../models/propTypes/image';
 import Figure from '../Figure';
+import Blocks from '../Blocks';
+import Fragment from '../Fragment';
+import InlineLink from '../InlineLink';
 
 const DEFAULT_IMAGE_RES = 640;
 
 const getText = ({ model }) => model.blocks[0].model.blocks[0].model.text;
 
+const captionComponentsToRender = { fragment: Fragment, urlLink: InlineLink };
 const getCaption = block => {
   if (!block) {
     return null;
   }
-  return getText(block);
+
+  return (
+    <Blocks
+      blocks={block.model.blocks[0].model.blocks[0].model.blocks}
```

Comment:
Could this be moved inside caption container? Doesn't seem very image specific and will have to be moved anyway when we have video